src/fe-text/terminfo-core.c

```diff
@@ -535,6 +537,11 @@ void terminfo_cont(TERM_REC *term)
 	if (term->TI_smkx)
 		tput(tparm(term->TI_smkx));
 
+	if (bracketed_paste_enabled) {
+		bracketed_paste_enabled = FALSE;
+		term_set_bracketed_paste_mode(TRUE);
```

Comment:
We shouldn't be messing with the tty settings so frequently but that's another problem, anyway that's not a problem since it's just a bunch of bytes and I'd vouch for simple and clear code instead